Components/Card/card.css

```diff
@@ -0,0 +1,335 @@
+  --horizontal-card-price-discount: #9a9a9a;
+  --horizontal-card-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+  --horizontal-card-bg: white;
+  --horizontal-card-CTA-border-color: black;
+  --checkout-card-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+  --color-gray: gray;
+  --color-black: black;
+  --color-white: white;
+  /* horizontal card end */
+  /* wishlist vertical card  */
+  --vertical-card-primary-color: purple;
+  --vertical-card-secondary-color: #ff527b;
+  --vertical-card-discount-color: #ff905a;
+  --vertical-card-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+  /* vertical card end */
+  /* products vertical cards */
+  --filer-bg: rgb(255, 255, 255);
+  --card-primary-color: purple;
+  --card-secondary-color: #ff527b;
+  --card-discount-color: #ff905a;
+  --badge-icon-color: #fb3958;
+  --color-white: white;
+  --color-black: black;
+  --ratings-bg: rgb(224, 202, 202);
+  --ratings-icon-color: green;
+  /*  products vertical cards end*/
+}
+
+/* basic card */
+
+.basic-card {
+  max-width: 21.875rem;
+  background-color: var(--color-white);
+  padding-bottom: 1rem;
+  border-radius: 0.5rem;
+  box-shadow: 1px 2px 2px var(--color-gray);
+}
+
+.basic-card-body {
+  padding: 0 1rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.basic-card-header > * {
+  margin: 0.4rem 0;
+}
+
+.basic-card-header > .basic-card-title {
+  margin-top: 1rem;
+}
+
+.basic-card-actions > .secondary-btn {
+  font-size: 1.4rem;
+  color: var(--color-gray);
+}
+
+.basic-card-intro {
+  font-weight: 400;
+}
+
+/* horizontal */
+
+.saiyan-horizontal-card {
+  display: flex;
+  gap: 2rem;
+  width: 600px;
+  box-shadow: var(--horizontal-card-shadow);
+  border-radius: 0.6rem;
+  background-color: var(--horizontal-card-bg);
+}
+
+.saiyan-horizontal-card .card__img__container {
+  width: 250px;
+  display: flex;
+  align-items: center;
+}
+
+.saiyan-horizontal-card .card__img {
+  width: 100%;
+}
+
+.saiyan-horizontal-card .card__quantity {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.saiyan-horizontal-card .card__quantity__box {
+  width: 30px;
+  height: 20px;
+  border-radius: 0.5rem;
+  border: none;
+  outline: none;
+  border: 1px solid var(--color-gray);
+  text-align: center;
+}
+
+.saiyan-horizontal-card .card__quantity__btn {
+  border: none;
+  border: 1px solid var(--color-gray);
+  border-radius: 50%;
+  font-size: 1rem;
+  font-weight: 600;
+}
+
+.saiyan-horizontal-card .card__CTA__container {
+  margin-top: 2rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.saiyan-horizontal-card .card__CTA__btn {
+  padding: 0.5rem 1rem;
+  /* border: 1px solid var(--horizontal-card-CTA-border-color); */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 0.6rem;
+  text-decoration: none;
+  font-weight: 600;
+  background-color: var(--card-primary-color);
+  color: var(--color-white);
+}
+
+.saiyan-horizontal-card .card__CTA__btn--secondary {
```

Comment:
`card__CTA__btn--secondary` you can re-name this to make it smaller & readable

Reply:
using this in ecommerce app. will fix it, if i get time.